lib/nifty_services.rb

```diff
@@ -14,6 +13,7 @@ module NiftyServices
   autoload :Error,              'nifty_services/errors'
   autoload :Errors,             'nifty_services/errors'
   autoload :Util,               'nifty_services/util'
+  autoload :Hash,               'nifty_services/support/hash'
```

Comment:
Can you move this to a simple `require 'nifty_services/support/hash'`(good folder name, btw :+1: ) in require section of this file? For me looks much more semantic, since  `Hash` inst coupled to `NiftyServices` namespace